WordPress/openverse
# Enable the isort rules for ruff and fix linting issues

## Fixes



Fixes #3695 by @stacimc 

## Description


When we moved from isort to ruff, the import sorting in the pre-commit step was broken. This PR:

* Updates `pyproject.toml` to enable the isort rules for ruff, which causes import sorting to begin working again in the pre-commit.
* Fixes all linting issues with imports that were merged while the pre-commit wasn't working.


## Testing Instructions


Pick any file and move the imports out of order (out of alphabetical order, change the spaces between import blocks, move local imports before third party, etc). Run `just lint` and verify it fails and updates the file. Verify that no other files were changed (all other fixes should be present in this PR).


## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
